js/faq.js

```diff
@@ -1,4 +1,19 @@
 const faqTitles = document.querySelectorAll('.faq__title');
+const faqLinks = document.querySelectorAll('.user_FAQ');
+
+var currentLocation = window.location.hash;
```

Comment:
any specific reason to go with `var`?

Reply:
no! we could have used let or const too!

Reply:
all checks have failed?

Reply:
Please check the details of the checks being failed and THEN read this: https://github.com/Real-Dev-Squad/website-welcome#scripts-used-in-project

Reply:
I see! I have formatted the code with prettier.